zcl_text2tab_parser.clas.abap

```diff
@@ -204,7 +211,9 @@ endmethod.  "apply_conv_exit
 method BREAK_TO_LINES.
   data:
         l_found type i,
+        idx TYPE i,
         l_break type string value c_crlf.
+  FIELD-SYMBOLS: <line> TYPE string.
```

Comment:
pls keep to formatting style of all other code. I mean lower cased key words. Same for all other code.
UPD: exception is the definitions, which are sometimes formatted on regeneration ...